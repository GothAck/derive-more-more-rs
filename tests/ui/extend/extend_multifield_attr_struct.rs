use derive_more_more::Extend;

#[derive(Default, Extend)]
#[extend(usize)]
struct Named {
    inner: Vec<usize>,
    test: bool,
}

#[derive(Default, Extend)]
#[extend(usize)]
struct Unnamed(Vec<usize>, bool);

fn main() {}
