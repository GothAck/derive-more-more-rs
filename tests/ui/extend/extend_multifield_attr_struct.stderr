error: Struct has more than one field, move the `extend` attribute to the appropriate field
 --> tests/ui/extend/extend_multifield_attr_struct.rs:4:1
  |
4 | #[extend(usize)]
  | ^

error: Struct has more than one field, move the `extend` attribute to the appropriate field
  --> tests/ui/extend/extend_multifield_attr_struct.rs:11:1
   |
11 | #[extend(usize)]
   | ^
